biome.jsonc

```
@@ -0,0 +1,86 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.8.3/schema.json",
+  "files": {
+    "include": ["./**/*.ts", "./**/*.json"],
+    "ignore": ["node_modules/", "bun.lockb", ".astro/"]
```

Since we're not using Astro in this project
```suggestion
    "ignore": ["node_modules/", "bun.lockb"]
```